documentation/modules/ref-kafka-connect-configuration.adoc

```diff
@@ -42,8 +42,10 @@ Certain options have default values:
 
 These options are automatically configured in case they are not present in the `KafkaConnect.spec.config` or `KafkaConnectS2I.spec.config` properties.
 
-Use the three allowed `ssl` configuration options for client connection using a specific _cipher suite_ for a TLS version.
+There are exceptions to the restrictions imposed on the configuration.
```

Comment:
_Suggested change:_

There are exceptions to the restrictions imposed on the configuration > There are exceptions to the forbidden options

Clarify that the exceptions are related to the list of "forbidden" options above. I was not quite sure at first.

Reply:
Updated in the 4 locations